[format] Fix issue using the '-y' flag (#3058)
demisto/demisto-sdk: CHANGELOG.md

* Updated the integration Traffic Light Protocol (TLP) color list schema in the **validate** command.
* Fixed an issue where **upload** would not read a repo configuration file properly.
* Fixed an issue where **upload** would not handle the `-x`/`--xsiam` flag properly.

> * Fixed an issue where **format** not working properly using the `-y`/`--assume-yes` flag on the json file.

Reviewer: ```suggestion
* Fixed an issue where **format** failed to use input from the user, when asking about a `from_version`.
```
